hathor/simulator/simulator.py

```diff
@@ -149,7 +150,8 @@ def create_peer(
         enable_sync_v2: bool = True,
         soft_voided_tx_ids: Optional[Set[bytes]] = None,
         full_verification: bool = True,
-        event_ws_factory: Optional[EventWebsocketFactory] = None
+        event_ws_factory: Optional[EventWebsocketFactory] = None,
+        tx_storage: Optional[TransactionStorage] = None
```

Comment:
Why do you need to set `tx_storage`? I was considering to refactor all tests to remove `Builder.set_storage()`. Can't you use `Builder.use_memory()` or `Builder.use_rocksdb()` instead?

Reply:
This has been removed after https://github.com/HathorNetwork/hathor-core/pull/653 was merged to master and I rebased it here

Reply:
Ok. Thanks!